iOS/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -151,7 +200,11 @@ extension CourseListViewController: UISearchResultsUpdating {
 
         self.collectionView?.setContentOffset(scrollOffset, animated: trueUnlessReduceMotionEnabled)
 
-        guard let searchText = searchController.searchBar.text, !searchText.isEmpty, searchController.isActive else {
+        let searchText = searchController.searchBar.text
+        let hasSearchText = !(searchText?.isEmpty ?? true)
```

Comment:
Isn't there is a clearer way of expressing that? Having the constant `true` that is being reverse to `false` in the expression can be misleading, I would prefer an explicit if there.

Reply:
Done